aviary/interface/methods_for_level2.py

```diff
@@ -2195,23 +2195,14 @@ def _add_guesses(self, phase_name, phase, guesses):
             guesses["altitude"] = ([initial_altitude, final_altitude], 'ft')
 
         if self.mission_method is HEIGHT_ENERGY:
-            # if time not in initial guesses, set it to the average of the initial_bounds and the duration_bounds
-            if 'time' not in guesses:
-                initial_bounds = wrapped_convert_units(
-                    self.phase_info[phase_name]['user_options']['initial_bounds'], 's')
-                duration_bounds = wrapped_convert_units(
-                    self.phase_info[phase_name]['user_options']['duration_bounds'], 's')
-                guesses["time"] = ([np.mean(initial_bounds[0]), np.mean(
-                    duration_bounds[0])], 's')
-
             # if time not in initial guesses, set it to the average of the initial_bounds and the duration_bounds
```

Comment:
I think that the second block of code is the one that should be removed. The first one adds "time" to the guesses, so the second block will neve execute.